platform/viewer/src/appExtensions/GenericRoutesTemplateModule/routesTemplateModule.js

```diff
@@ -28,16 +28,16 @@ const ViewerLocalFileData = asyncComponent(() =>
   )
 );
 
-const ROUTES_DEF = {
-  viewer: {
+const ROUTES_DEF = [
```

Comment:
Where a definition is a a template name (string), template (component), and an optional condition (that expects to receive the appConfig)

Reply:
ok changed it to
{
name: String,
template: Component,
condition?
}